react/features/toolbox/components/buttons/AudioMuteButton.web.js

```diff
@@ -122,6 +123,19 @@ export class AudioMuteButton extends AbstractAudioMuteButton {
 
     _doToggleAudio: () => void;
 
+    /**
+     * Emits an event to signal audio mute should be toggled.
+     *
+     * @private
+     * @returns {void}
+     */
+    _doToggleAudio() {
```

Comment:
Since the method is overriding the logic in the abstract class won't be executed. Is this intentional?

Reply:
Yes, it's intentional. I just need to get this fixed quickly for deploy and this was the most straightforward shortcut I could see. I don't know what I'm going to do for the real fix. Ideally web and native could use the same flows but I didn't realize the special cases handled in conference.js.

Reply:
Sure! I know. I just wanted to verify that it is intentional!